style.css

```diff
@@ -583,10 +622,11 @@ textarea {
     width: 100%;
   }
 
- .about-description {
+  .about-description {
     font-size: 15px;
-  } 
+  }
 
+  /* Contact section */
   .contact-me {
     background-image: url(assets/images/desktop-bg-img.png);
   }
```

Comment:
- [ ] Contact section has a lot of padding on top but almost none at the bottom, they should be evened out, kindly do that, please use the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848) to design how much to add for the top and for the bottom. 😁

This is your design right now:

![image](https://user-images.githubusercontent.com/20567503/178044667-8af39aa7-902e-4392-a253-b4f2e2b7b2c4.png)

And this is the Figma design:

![image](https://user-images.githubusercontent.com/20567503/178044892-aec15051-6bf3-40b5-81a9-ed0cb126a6b8.png)

Reply:
@mirouhml Thank you for this review.